### ISSUE TITLE
Alert History failing to load 

### ISSUE THREAD

**Issue by karel-rehor** [influxdata/chronograf#3437]
**nightly build**

Build time **n201805140829**

Browser **Chrome 66**

```
$ chronograf --version
2018/05/14 14:51:23 Chronograf 1.2.0~n201805140829 (git: c55276ccb46f4e0f2eda4414e1915a139eac5304)

```

*Use Case*

1) Create an alert in the alert builder guaranteed to throw an alert (e.g. cpu.usage_idle < 100) 
2) Wait a short while for an alert to be generated then open Alert History 

The page never seems to load 

3) Open the status page 

Here too the Alert History panel never seems to finish loading. 

4) In data explorer check the contents of chronograf.autogen.alerts - alerts are being written to the db

*Expected Behavior*

Alert history list should load 

*Screenshots* 

![alerthistorynotloading01](https://user-images.githubusercontent.com/25881301/39998851-af5c2920-5787-11e8-9e41-089dc4e30238.png)
![alerthistorynotloading02](https://user-images.githubusercontent.com/25881301/39998852-af768374-5787-11e8-8b7c-2f38ed74cd59.png)
![alerthistorynotloading03](https://user-images.githubusercontent.com/25881301/39998854-af8fd0e0-5787-11e8-9e4a-31c1108be485.png)
![alerthistorynotloading04](https://user-images.githubusercontent.com/25881301/39998855-afa9840e-5787-11e8-9247-55b5d2cd7ecc.png)
![alerthistorynotloading05](https://user-images.githubusercontent.com/25881301/39998856-afc2dc74-5787-11e8-975e-bca912300064.png)

Nothing noticed yet in console or other logs.

**Comment by bthesorceror** [comment]
@karel-rehor can you confirm what version of Kapacitor that you are running? I am not able to reproduce locally and want to make sure I have the same version as you.

**Comment by russorat** [comment]
@bthesorceror i saw this on current master this morning. i'll help repro.

**Comment by bthesorceror** [comment]
Closed with #3445

**Comment by russorat** [comment]
great!
